CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Unreleased
 * **Removed**
+   * REMOVED: unused headers [#4829](https://github.com/valhalla/valhalla/pull/4829)
```

Comment:
we do the changelog entries such that new ones are at the bottom of the current unreleased section